### DISCUSSION TITLE
Released v4.3!

### DISCUSSION THREAD

**Discussion by pit-ray** [pit-ray/win-vind#52] [News]
Today, I released win-vind v4.3!🎉

https://github.com/pit-ray/win-vind/releases/tag/v4.3.0

It now has the cmd2cmd macro function to generate keystrokes and the ability to load a remote .vindrc.
There are also a few other useful changes.

Thanks to all of you for support. Feedback is welcome.

Demo

https://user-images.githubusercontent.com/42631247/147887231-24424d36-410d-4379-8846-7dc6a5a7f355.mp4

**Comment by pit-ray** [comment]
Fixed some bugs.
https://github.com/pit-ray/win-vind/releases/tag/v4.3.1

**Comment by pit-ray** [comment]
Supported French keyboard layout.
https://github.com/pit-ray/win-vind/releases/tag/v4.3.2

**Comment by gilbh** [comment]
Do you expect approval from Chocolatey for recent version? Should I wait for Chocolatey approval or switch ti winget?

Thanks for all the hard work!

**Reply by pit-ray** [reply]
I submitted 4.3 to Chocolatey. (ref. https://community.chocolatey.org/packages/win-vind#versionhistory)  
It passed the automatic checks, but it is not reviewed by any human yet. It used to be approved within a few hours.

You can update it manually instead, as described in the README.

> However, [win-vind 4.3.2](https://community.chocolatey.org/packages/win-vind/4.3.2) is not still approved on Chocolatey. Therefore, download the [Chocolatey package source](https://github.com/pit-ray/win-vind/releases/download/v4.3.2/win-vind_4.3.2_chocolatey_package_source.zip) manually and execute the following command in the extracted directory.
> 
> ```sh
> $ cd win-vind_4.3.2_chocolatey_package_source
> $ choco install win-vind.4.3.2.nupkg
> ```

**Comment by gilbh** [comment]
Yes, I've seen your instructions but wanted an idea on when the Chocolatey installation will be ready and whether I should wait for it ... I guess that if a new release has features that are specifically important to me I'd take the quick route, otherwise wait for Chocolatey.

Thanks!

**Reply by pit-ray** [reply]
Chocolatey is now available.

**Reply by pit-ray** [reply]
Created a status thread for winget(#68) and Chocolatey(#67). From now on, you can enable notifications in these threads to receive notifications.

**Comment by gilbh** [comment]
Also: I just used your installer (the "standard" Windows way) and it worked great. So it's important to remember there is also this option ...

**Reply by pit-ray** [reply]
winget calls the installer version internally, so it is recommended if you want to do version control. Or use the update feature in the task tray.

**Comment by gilbh** [comment]
I am having trouble running win-vind v4.3 on my laptop. It had a chocolatey installation of v4.2 (separate), but I am not sure if that is the reason. In any case, for both Installer and Portable 64bit, I am getting the following error trying to run the exe file:
![image](https://user-images.githubusercontent.com/39532610/148703618-93771ef4-1a69-445c-97e9-98b2569ee6e5.png)

**Reply by pit-ray** [reply]
It's most likely a bug, but please use Issue.

**Comment by pit-ray** [comment]
Fixed an issue that prevented from starting before Windows 10 2004.
https://github.com/pit-ray/win-vind/releases/tag/v4.3.3
